src/commands/admin/TestCommand.js

```diff
@@ -2,552 +1,57 @@
+const CT = require("../../core/CommandsTest");
+CT.init().then();
 
-const DraftBotReactionMessageBuilder = require("../../core/messages/DraftBotReactionMessage").DraftBotReactionMessageBuilder;
-const DraftBotReaction = require("../../core/messages/DraftBotReaction").DraftBotReaction;
+module.exports.help = {
+	name: "test"
+};
 
-const Maps = require("../../core/Maps");
 /**
  * Cheat command for testers
  * @param {("fr"|"en")} language - Language to use in the response
  * @param {module:"discord.js".Message} message - Message from the discord server
  * @param {String[]} args=[] - Additional arguments sent with the command
  */
-const TestCommand = async (language, message, args) => {
-	let authorized = false;
 
+const TestCommand = async(language, message, args) => {
```

Comment:
Codacy has a fix for the issue: [Missing space before function parentheses.](https://app.codacy.com/gh/DraftBot-A-Discord-Adventure/DraftBot/pullRequest?prid=7613090)
```suggestion
const TestCommand = async (language, message, args) => {
```